lib/MusicBrainz/Server/Edit/Series/Edit.pm

```diff
@@ -82,6 +89,8 @@ sub build_display_data {
 around allow_auto_edit => sub {
     my ($orig, $self, @args) = @_;
 
+    return 1 if $self->can_amend($self->series_id);
```

Comment:
No, it doesn’t, it triggers an ISE as it always did. That should probably be fixed apart.